media-fonts/fonts-meta/fonts-meta-2.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Meta package for fonts to cover most needs"
+HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cjk emoji +latin ms"
+
+LANGS="as bn bo brx doi dz gu hi km kn kok ks mai ml mr ne or pa ru sa sat sd si syc ta te th"
+for lang in ${LANGS}; do
+	IUSE+=" l10n_${lang}"
+done
+unset lang LANGS
+
+RDEPEND="media-fonts/cantarell
+	media-fonts/droid
+	media-fonts/noto
+	media-fonts/oldstandard
+	media-fonts/powerline-symbols
+	media-fonts/ubuntu-font-family
+	media-libs/fontconfig
+	|| (
+		media-fonts/source-pro
+		media-fonts/anonymous-pro
+		media-fonts/cascadia-code
+		media-fonts/fantasque-sans-mono
+		media-fonts/fira-code
+		media-fonts/fira-mono
+		media-fonts/fira-sans
```

Comment:
The package description says this is a monospace font, but it doesn't look like it...
https://mozilla.github.io/Fira/
```suggestion
```